app/views/api/tasks/index.jbuilder

```diff
@@ -1,30 +1,30 @@
 json.tasks do
   json.array! @tasks do |task|
-    json.id         @task.id
-    json.content    @task.content
-    json.completed  @task.completed
-    json.due        @task.due
-    json.created_at @task.created_at
-    json.updated_at @task.updated_at
+    json.id         task.id
```

Comment:
You should be referring to the `task` from the loop `|task|`, not `@task`, which is an instance variable.